pkg/cmd/server/apis/config/validation/master.go

```diff
@@ -212,9 +213,13 @@ func ValidateAuditConfig(config configapi.AuditConfig, fldPath *field.Path) Vali
 		return validationResults
 	}
 
+	// TODO (soltysh): both of these should be turned into errors in 3.11
 	if len(config.AuditFilePath) == 0 {
 		// for backwards compatibility reasons we can't error this out
 		validationResults.AddWarnings(field.Required(fldPath.Child("auditFilePath"), "audit can not be logged to a separate file"))
```

Comment:
isn't this actually phrased in the other direction?  You're requiring it to be logged into a separate file?

Reply:
Yeah, it should be a separate file atm. Will fix right away.